iModelCore/iModelPlatform/DgnCore/DgnModel.cpp

```diff
@@ -821,7 +821,8 @@ BeGuid GeometricModel::QueryGeometryGuid() const {
     stmt.BindId(1, m_modelId);
     stmt.Step();
 
-    return stmt.GetValueGuid(0);
+    auto guid = stmt.GetValueGuid(0);

```

Comment:
This looks like a change that was made for debugging purposes. I don't mind leaving it in if desired, but you may not have intended to commit it.

Reply:
Reverted.